ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml

```diff
@@ -135,7 +135,7 @@ objects:
         echo "[INFO] Leased resource: ${resource}"
 
         function release() {
-            local resource_name; resource_name="$( jq .name <<<"${resource}" --raw-output )"
+            local resource_name; resource_name="$( jq .name --raw-output <<<"${resource}" )"
```

Comment:
still has the option to the right of the `.name` positional argument, but whatever ;)

/lgtm